release/cli/pkg/operations/bundle_release.go

```diff
@@ -146,3 +149,22 @@ func GenerateBundleSpec(r *releasetypes.ReleaseConfig, bundle *anywherev1alpha1.
 	fmt.Printf("%s Successfully generated bundle manifest spec\n", constants.SuccessIcon)
 	return nil
 }
+
+func getImageDigest(_ context.Context, r *releasetypes.ReleaseConfig, artifact releasetypes.Artifact) (string, error) {
```

Comment:
Is the `context.Context` param here needed?

Reply:
Responded [above](https://github.com/aws/eks-anywhere/pull/7186#discussion_r1433300878)